### DISCUSSION TITLE
Embedded spaces in script parameters?

### DISCUSSION THREAD

**Discussion by sig-daveka** [Camelcade/Perl5-IDEA#2388] [Q&A]
How do I enter embedded spaces in Run / Debug script parameters?  When invoking my script from the command line I'd do something like `--since '20 days'`, but when entering that as a script parameter I get passed `'20` and `days'` as separate arguments, complete with quotes.

**Comment by hurricup** [comment]
ye, encountered this myself yesteday :(

**Reply by hurricup** [reply]
Created #2389

**Reply by hurricup** [reply]
There is no workaround currently available. Still this is fixed in respective issue. Feel free to use a plugin from CI when it's ready:

- For 2021.1 https://github.com/Camelcade/Perl5-IDEA/actions/runs/949913748
- For 2021.2 EAP https://github.com/Camelcade/Perl5-IDEA/actions/runs/949899298

Note, that you should quote such arguments with double quotes. Let me know if it helps.
